src/scheduler.py

```diff
@@ -37,32 +39,47 @@
     "fedring": [FedRingServer,FedRingClient],
     "swarm" : [SWARMServer, SWARMClient],
     "dispfl": [DisPFLServer, DisPFLClient],
-    "defkt": [DefKTServer,DefKTClient],
+    "defkt": [DefKTServer, DefKTClient],
     "fedfomo": [FedFomoServer, FedFomoClient],
-    "l2c": [L2CServer,L2CClient],
-    "metal2c": [MetaL2CServer,MetaL2CClient],
+    "l2c": [L2CServer, L2CClient],
+    "metal2c": [MetaL2CServer, MetaL2CClient],
     "centralized": [CentralizedServer, CentralizedCLient],
     "feddatarepr": [FedDataRepServer, FedDataRepClient],
     "fedval": [FedValServer, FedValClient],
 }
 
+
 def get_node(config: dict, rank) -> BaseNode:
     algo_name = config["algo"]
-    return algo_map[algo_name][rank>0](config)
+    return algo_map[algo_name][rank > 0](config)
 
 
 class Scheduler():
     """ Manages the overall orchestration of experiments
     """
+
     def __init__(self) -> None:
         pass
 
-    def assign_config_by_path(self, config_path) -> None:
-        self.config = load_config(config_path)
-        
     def install_config(self) -> None:
         self.config = process_config(self.config)
 
+    def assign_config_by_path(self, sys_config_path, algo_config_path):
+        self.sys_config = load_config(sys_config_path)
+        self.algo_config = load_config(algo_config_path)
+        self.merge_configs()
+
+    def merge_configs(self):
+        self.config = self.algo_config.copy()
+        self.config.update({
+            "dset": "cifar10",
```

Comment:
can you remove this hardcoding? We do not want any parameter being hardcoded inside scheduler.py
Only the config needs to be the single source of truth.